docs/ops/arithmetic/Log_1.md

```diff
@@ -6,6 +6,12 @@
 
 **Short description**: *Log* performs element-wise natural logarithm operation with given tensor.
 
+**Detailed description**: *Log* does the following with the input tensor *a*:
+
+\f[
+a_{i} = log(a_{i})
+\f]
+
 **Attributes**:
 
     No attributes available.
```

Comment:
**Inputs**
* **1**: An tensor of type T and arbitrary shape. **Required.**  

**Outputs**
* **1**: The result of element-wise log operation. A tensor of type T and the same shape as input.